stand-alone/yarn.lock

```diff
@@ -4697,6 +4692,22 @@ styled-components@^5.0.1:
     shallowequal "^1.1.0"
     supports-color "^5.5.0"
 
+styled-components@^5.3.3:
```

Comment:
im a newbie, but i thought that we've deleted styled components

Reply:
Nice observation. The `package.json` file remains unchanged, so I'm curious about the alterations in the lock file. 🤔

Reply:
Damn, I might've push it accidentally. Thanks guys for catching! Let me look into it...